pg_yregress/test.c

```diff
@@ -4,6 +4,8 @@
 
 #include "pg_yregress.h"
 
+#include "uthash.h"
```

Comment:
`uthash` looks stable and small. Given you are looking at deprecating STC, I am leaving it as is for now.

Reply:
Not looking at deprecating quite yet. What I am trying to avoid is the sprawl of libraries that largely do the same things. Not an immediate concern for the PR itself, but this sprawl is not great to have.

Reply:
understand & agree. Keeping it as is for now.

Reply:
See [this comment](https://github.com/omnigres/omnigres/pull/467#issuecomment-1911461969)